test/rotateArray.test.js

```diff
@@ -0,0 +1,28 @@
+const { rotateArray } = require("../lib");
+
+describe("rotateArray", () => {
+  test("rotateArray([1,2,3], 2) == [2, 3, 1]", () => {
```

Comment:
The test descriptions are more useful as a written statement. something like "Rotates items in array" and "Returns identity for a 1 or 0 length array" "Mutates given array" or "Does not mutate array", etc.